src/controllers/routeActivation.ts

```diff
@@ -0,0 +1,105 @@
+import { Context, Next } from 'koa';
+import dotenv from 'dotenv';
+import { ProcessorTransformationRequest, RouterTransformationRequest } from '../types';
+
+dotenv.config();
+
+const transformerMode = process.env.TRANSFORMER_MODE;
+const startDestTransformer = transformerMode === 'destination' || !transformerMode;
+const startSourceTransformer = transformerMode === 'source' || !transformerMode;
+const transformerDelivery = process.env.TRANSFORMER_DELIVERY || true;
+const deliveryTestModeEnabled =
+  process.env.TRANSFORMER_DELIVERY_TEST_ENABLED?.toLowerCase() === 'true' || false;
+const transformerTestModeEnabled = process.env.TRANSFORMER_TEST_MODE
+  ? process.env.TRANSFORMER_TEST_MODE.toLowerCase() === 'true'
+  : false;
+const areFunctionsEnabled = process.env.ENABLE_FUNCTIONS === 'false' ? false : true;
+const destinationFilterList = process.env.DESTINATION_FILTER_LIST?.toLocaleLowerCase();
+const sourceFilteList = process.env.SOURCE_FILTER_LIST?.toLocaleLowerCase();
+const deliveryFilterList = process.env.DESTINATION_DELIVERY_FILTER_LIST?.toLocaleLowerCase();
+
+export default class RouteActivationController {
+  private static executeActivationRule(ctx: Context, next: Next, condition: boolean) {
+    if (condition) {
```

Comment:
I'd rename the name of variable `condition` as `dontActivateRoute`
Or 
For easiness, I'd replace condition itself in below way

```javascript
// In this way the caller should change the condition
private static executeActivationRule(ctx: Context, next: Next, shouldActivate: boolean) {
    if (shouldActivate) {
      return next();
    }
    ctx.status = 404;
    ctx.body = 'RouteActivationController route is disabled';
    return ctx;
  }
````

But this can be written other way round as well
```javascript
private static executeActivationRule(ctx: Context, next: Next, dontActivate: boolean) {
    if (!dontActivate) {
      return next();
    }
    ctx.status = 404;
    ctx.body = 'RouteActivationController route is disabled';
    return ctx;
  }

// OR

private static executeActivationRule(ctx: Context, next: Next, shouldActivate: boolean) {
    if (dontActivate) {
       ctx.status = 404;
      ctx.body = 'RouteActivationController route is disabled';
      return ctx;
    }
    return next();
  }
```

Reply:
Addressed